test/jdk/java/io/File/MaxPathLength.java

```diff
@@ -193,14 +196,14 @@ static void testLongPath(int max, String fn,
         }
     }
 
-    private static void testLongPathOnWindows () throws Exception {
-        if (isWindows) {
-            String name = fileName;
-            while (name.length() < MAX_LENGTH) {
-                testLongPath (20, name, false);
-                testLongPath (20, name, true);
-                name = getNextName(name);
-            }
+    private static void testLongPath () throws Exception {
+        String name = fileName;
+        while (name.length() < MAX_LENGTH) {
+            testLongPath(MAX_PATH_COMPONENTS_WINDOWS, name, false);
+            testLongPath(MAX_PATH_COMPONENTS_WINDOWS, name, true);
+            name = getNextName(name);
         }
     }
 }
+
```

Comment:
Looks like there are a couple of blank lines at the end. I thought whitespace detection would flag it.